stellargraph/link_prediction.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is currently at top-level, but happy to move it to wherever seems more appropriate